server/test/follow.spec.js

```diff
@@ -0,0 +1,149 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+
+chai.use(chaiHttp);
+
+let userToken;
+
+describe('Follow/Unfollow Test Feature', () => {
+  describe('When User login', () => {
+    const payload = {
+      user: {
+        username: 'ugochukwu',
+        password: 'password'
+      }
+    };
+    beforeEach((done) => {
+      chai.request(app)
+        .post('/api/users/login')
+        .send(payload)
+        .end((err, res) => {
+          userToken = res.body.user.token;
+          if (err) return done(err);
+          done();
+        });
+    });
+    it('Should enable User to follow an author', (done) => {
+      chai.request(app)
+        .post('/api/profiles/joeeasy/follow')
+        .set('authorization', userToken)
+        .end((err, res) => {
+          expect(err).to.equal(null);
+          expect(res.status).to.equal(200);
+          expect(res.body.message).to.equal('You are now following joeeasy');
+          if (err) return done(err);
+          done();
+        });
+    });
+    it('Should not enable User to follow an author twice', (done) => {
```

Comment:
`Should not enable User to follow an author twice`  Replace `enable` with `allow`

Reply:
it has been replaced